lib/pengine/unpack.c

```diff
@@ -2687,15 +2690,15 @@ unpack_migrate_to_success(pe_resource_t *rsc, pe_node_t *node, xmlNode *xml_op,
         if (target_node && target_node->details->online) {
             pe_rsc_trace(rsc, "Marking active on %s %p %d", target, target_node,
                          target_node->details->online);
-            native_add_running(rsc, target_node, data_set);
+            native_add_running(rsc, target_node, data_set, action_fail_ignore);
         }
 
     } else { // Pending, or complete but erased
         if (target_node && target_node->details->online) {
             pe_rsc_trace(rsc, "Marking active on %s %p %d", target, target_node,
                          target_node->details->online);
 
-            native_add_running(rsc, target_node, data_set);
+            native_add_running(rsc, target_node, data_set, action_fail_ignore);
```

Comment:
In this case, I think action_fail_ignore is fine, because we do not know if there is a failure (the action is likely pending, and counting it toward the node priority could make sense).

Reply:
All right!